tests/acceptance/__init__.py

```diff
@@ -265,5 +267,6 @@ def run_gitlabform(config, target, include_archived_projects=True, noop=False):
         config_string=config,
         target=target,
         noop=noop,
+        output_file=output_file,
```

Comment:
Can we add a generic test case to validate this. Let's try to assert that the generated file is valid. I guess it should be okay as long as it's a valid yaml file?

Reply:
Thanks for adding the additional test.